py-rse/automate.Rmd

```diff
@@ -188,9 +123,9 @@ In this case,
 one of three things will happen:
 
 1.  If `results/moby_dick.csv` doesn't exist,
-    Make runs the action to create it.
+    Make runs the recipe to create it.
 2.  If `data/moby_dick.txt` is newer than `results/moby_dick.csv`,
-    Make runs the action to update the results.
+    Make runs the recipe to update the results.
 3.  If `results/moby_dick.csv` is newer than its prerequisite,
     Make does nothing.
 
```

Comment:
Should you show the output here (`results/mody_dick.csv` doesn't exist so it will create it)? I think it's still good to explain all the possibilities, but it looks kind of weird to run `$ make` and not show the output.

Reply:
I take that back - `results/moby_dick.csv` will exist already because it's one of the three result files generated in #325.

Reply:
Agree, fixed.